src/components/Card/Card.tsx

```diff
@@ -1,26 +1,24 @@
 import React from 'react';
 import './Card.css';
-import { Link } from 'react-router-dom';
 import veggies from '../../images/shelley-pauls.jpg';
 
 interface CardProps {
   id: number;
   distance: number;
   name: string;
+  findSelectedMarket: (marketID: number) => void;
 }
 
-export const Card: React.FC<CardProps> = ({ id, name, distance }) => {
+export const Card: React.FC<CardProps> = ({ id, name, distance, findSelectedMarket }) => {
   return (
-    <Link to={`/markets/${id}`} className='link'>
-      <article className='market-card' id={id.toString()}>
-        <div className='veggie-container'>
-          <img src={veggies} alt='fresh produce' className='veggie-img' />
-        </div>
-        <div className='market-info'>
-          <p>{name}</p>
-          <p>{distance} miles away</p>
-        </div>
-      </article>
-    </Link>
+    <article className='market-card' id={`${id}`} onClick={() => findSelectedMarket(id)}>
```

Comment:
Can you refactor to make this a Link again? (see comment above about history.push)